src/trans/gpu/internal/ledir_mod.F90

```diff
@@ -248,10 +248,11 @@ SUBROUTINE LEDIR(ALLOCATOR,ZINPS,ZINPA,ZINPS0,ZINPA0,ZOUT,ZOUT0,POA1,KF_FS)
 #ifdef OMPGPU
 #endif
 #ifdef ACCGPU
-#ifdef _CRAYFTN
-    !$ACC PARALLEL LOOP COLLAPSE(2) PRIVATE(KM,IA,J) FIRSTPRIVATE(KF_FS,IOUT_STRIDES0,IOUT0_STRIDES0) DEFAULT(NONE)
+    !$ACC PARALLEL LOOP COLLAPSE(2) PRIVATE(KM,IA,J) FIRSTPRIVATE(KF_FS,IOUT_STRIDES0,IOUT0_STRIDES0) DEFAULT(NONE) &
+#ifndef _CRAYFTN
+    !$ACC& ASYNC(1)
 #else
-    !$ACC PARALLEL LOOP COLLAPSE(2) PRIVATE(KM,IA,J) FIRSTPRIVATE(KF_FS,IOUT_STRIDES0,IOUT0_STRIDES0) DEFAULT(NONE) ASYNC(1)
+    !$ACC&
```

Comment:
Is this necessary in order to terminate the `&` on the preceding line?

Reply:
unfortunately yes.